experimental/dds/tree2/src/feature-libraries/flex-tree/treeEvents.ts

```diff
@@ -23,11 +23,44 @@ import { FlexTreeNode } from "./flexTreeTypes";
  *
  * @alpha
  */
-export interface TreeEvent {
+export interface ITreeEvent {
 	/**
 	 * The node of the tree where the listener receiving the event is attached.
 	 */
 	readonly target: FlexTreeNode;
+
+	/**
+	 * When called, prevents this event from bubbling up to the parent node.
+	 */
+	stopPropagation(): void;
+}
+
+/**
+ * {@inheritDoc ITreeEvent}
+ *
+ * @alpha
```

Comment:
Do we need to export this class anymore?

Reply:
Yes from the file, we instantiate it in `lazyNode.ts`, but it's not exported from the package. I'll make it `@internal` instead of `@alpha` though.